docs/source/examples/models/gpt-oss-sft.md

```diff
@@ -0,0 +1 @@
+../../../../llm/gpt-oss-sft/README.md
```

Comment:
We can have it here instead: https://docs.skypilot.co/en/latest/examples/training/index.html